publications.rkt

```diff
@@ -51,9 +51,21 @@
 @(define ARRAY "Workshop on Libraries, Languages, and Compilers for Array Programming (ARRAY)")
 @(define ESECFSE "European Software Engineering Conference and Symposium on the Foundations of Software Engineering (ESEC/FSE)")
 @(define CSUR "ACM Computing Surveys (CSUR)")
+@(define JAR "Journal of Automated Reasoning (JAR)")
+@(define TECS "ACM Transactions on Embedded Computing Systems (TECS)")
 
 @(define publications
    (list
+    (publication "Verifying a Concurrent Garbage Collector with a Rely-Guarantee Methodology"
+                 "Yannick Zakowski, David Cachera, Delphine Demange, Gustavo Petri, David Pichardie, Suresh Jagannathan, and Jan Vitek"
+                 TECS
```

Comment:
`TECS` should be `JAR`

The `TECS` paper is "Can Android Run on Time?"

Reply:
Oh, my, I spoiled is heavily. Hope 7f41bf3 deals with this.